core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -434,7 +434,7 @@ class SparkContext(config: SparkConf) extends Logging {
     _statusTracker = new SparkStatusTracker(this, _statusStore)
 
     _progressBar =
-      if (_conf.get(UI_SHOW_CONSOLE_PROGRESS) && !log.isInfoEnabled) {
+      if (_conf.get(UI_SHOW_CONSOLE_PROGRESS)) {
```

Comment:
Hi, @ankuriitg . Can we have a test case to prevent a regression?

Reply:
Sure, I just updated the test case